src/components/dialogs/export-dialog.js

```diff
@@ -38,24 +119,51 @@ const ExportDialog = ({
     title,
 }) => {
     const [availableFormats, setAvailableFormats] = React.useState('');
+    const [formatsWithParameters, setFormatsWithParameters] = useState([]);
     const [selectedFormat, setSelectedFormat] = React.useState('');
     const [loading, setLoading] = React.useState(false);
-    const [downloadUrl, setDownloadUrl] = React.useState('');
     const [exportStudyErr, setExportStudyErr] = React.useState('');
 
+    const [checked, setChecked] = React.useState(false);
```

Comment:
Could we have a more specific state name. It doesn't make sense to have a `dialog` checked or not.

Reply:
You are right. developed / opened / unfolded ?